### PR TITLE
Add kubernetes helm chart

### PR DESCRIPTION
**Pull request by TheChef23**
In this PR i'd like to offer the helm chart I have built (https://github.com/TheChef23/helm-librenms) to be added into the main repo for other kubernetes users to use. What changes would be needed for this to get merged into the main brach? Or perhaps the chart is not wanted in this repo, if so I apologize. Is there another place I could offer this chart?

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [X] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: charts/librenms/values.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  # Custom configuration options for LibreNMS. Fore more infomation on options
+  # in this file check the following link:
+  # https://docs.librenms.org/Support/Configuration/
+  configuration: |-
+    $config['distributed_poller_group']          = '0';
+    $config['distributed_poller']                = true;
+  
+  # Laravel appkey used by LibreNMS, a default is set for useability it is
+  # recommended to replace this with your own key
+  appkey: base64:Qh1eBwRfpgNHWByDDqhaX0I/0hnE/VDRiLHLnjwrZRw=
```
Comment (murrant) [inline charts/librenms/values.yaml:23]: This should not have  a valid APP_KEY, users will just paste it in.

File: charts/librenms/files/custom.php.tpl [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?php
```
Comment (murrant) [inline charts/librenms/files/custom.php.tpl:1]: yaml configuration is preferred

**Comment by murrant** [comment]
A few notable changes.  Not sure about merging this upstream unless someone is able to maintain it.

**Comment by TheChef23** [comment]
Thanks for the feedback! I'll check those out. I run this in production and if you'll allow me and someone would be able to tell me what would be expected of me, i'd be willing to maintain the chart.

**Comment by murrant** [comment]
@TheChef23 just updated it when needed. Is there anyway the charts folder could be not in the top level or does it need to be there? We already have a lot of clutter there :)

**Comment by Jellyfrog** [comment]
If we want to maintain a offical chart it should go into its own repo I think.

File: charts/librenms/values.yaml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+# Declare variables to be passed into your templates.
+
+
+librenms:
+  image:
+    repository: librenms/librenms
+    tag: 24.5.0
+
+  # Timezone used by librenms for communication with RRD cached
+  timezone: UTC
```
Comment (Jellyfrog) [inline charts/librenms/values.yaml:12]: You have two settings for timezone.

**Comment by TheChef23** [comment]
With the chart being in it's own repo, it would also allow for a more simple helm chart release pipeline git GH pages as the helm repo. It would also allow for rennovate to suggest updates when needed, so when LibreNMS docker image gets released, we get a pull request.

See the original repo I released this chart in:
https://github.com/TheChef23/helm-librenms

**Comment by murrant** [comment]
I can make a new repo for it.  Would it be librenms/helm or librenms/librenms-helm?

**Comment by Jellyfrog** [comment]
> I can make a new repo for it. Would it be librenms/helm or librenms/librenms-helm?

Chart or charts id say

**Comment by TheChef23** [comment]
I would go with `librenms/helm-charts`, this is explicit about it being solely for Helm. It would then have a directory structure like:
```
charts/librenms
charts/rrdcached
```

This allows for multiple charts to be added to this repo in the future, for example, we could decide that rrdcached should have it's own charts which we import as dependency. For now it would only be LibreNMS.

**Comment by TheChef23** [comment]
@Jellyfrog if you could create the repo, i'd be happy to do the initial setup of pipelines and such.

**Comment by murrant** [comment]
@TheChef23 repo already created, you should be able to push to it.

**Comment by TheChef23** [comment]
We talked an discord and I'm able to work on the new repo:
https://github.com/librenms/helm-charts

I'll get some stuff set up and let you guys know. I'll close this PR now.

**Pull request closed without merging**